dev-tools/build-packages/wazuh-dashboard.Dockerfile

```diff
@@ -1,50 +1,24 @@
-# Usage: docker build --build-arg NODE_VERSION=18.19.0 --build-arg WAZUH_DASHBOARDS_BRANCH=4.10.0 --build-arg WAZUH_DASHBOARDS_PLUGINS=4.10.0 --build-arg WAZUH_SECURITY_DASHBOARDS_PLUGIN_BRANCH=4.10.0 --build-arg OPENSEARCH_DASHBOARDS_VERSION=2.13.0 -t wzd:4.10.0 -f wazuh-dashboard.Dockerfile .
+# Usage: docker build --build-arg NODE_VERSION=18.19.0 --build-arg WAZUH_DASHBOARDS_BRANCH=4.10.0 --build-arg WAZUH_DASHBOARDS_REPORTING_BRANCH=4.10.0 --build-arg WAZUH_DASHBOARDS_PLUGINS=4.10.0 --build-arg WAZUH_SECURITY_DASHBOARDS_PLUGIN_BRANCH=4.10.0 --build-arg OPENSEARCH_DASHBOARDS_VERSION=2.13.0 -t wzd:4.10.0 -f wazuh-dashboard.Dockerfile .
 
 ARG NODE_VERSION
 FROM node:${NODE_VERSION} AS base
 ARG OPENSEARCH_DASHBOARDS_VERSION
 ARG WAZUH_DASHBOARDS_BRANCH
 ARG WAZUH_DASHBOARDS_PLUGINS
 ARG WAZUH_SECURITY_DASHBOARDS_PLUGIN_BRANCH
+ARG WAZUH_DASHBOARDS_REPORTING_BRANCH
 ENV OPENSEARCH_DASHBOARDS_VERSION=${OPENSEARCH_DASHBOARDS_VERSION}
 USER root
 RUN apt-get update && apt-get install -y git zip unzip curl brotli jq
 USER node
-RUN git clone --depth 1 --branch ${WAZUH_DASHBOARDS_BRANCH} https://github.com/wazuh/wazuh-dashboard.git /home/node/wzd
-RUN chown node.node /home/node/wzd
 
-WORKDIR /home/node/wzd
-RUN yarn osd bootstrap --production
-RUN yarn build --linux --skip-os-packages --release
+COPY ./config-dockerfile/entrypoint.sh /entrypoint.sh
+ENTRYPOINT ["/usr/local/bin/entrypoint.sh"]
```

Comment:
**question**: is there a file in that location? I do not see that a file is copied or it could exist.

Reply:
done in the [commit](https://github.com/wazuh/wazuh-dashboard/pull/340/commits/a2bf704356bbd220c975fc5f381788f012fe85f6)

![image](https://github.com/user-attachments/assets/3a515576-dc8c-4b33-b3e3-68f26d510610)